### ISSUE TITLE
Server for verifiers of credentials/presentations

### ISSUE THREAD

**Issue by sgreenbury** [alan-turing-institute/trustchain#56]
The presentation counterpart of #55.

Implement a frontend and backend servers for verifiers to receive VCs/VPs from holders based upon [credible process](https://github.com/spruceid/credible#verifiablepresentationrequest) process and [demonstration](https://demo.spruceid.com/).

Aim to use the following crates:
- [actix-web](https://crates.io/crates/actix-web): for backend handling http requests
- [yew](https://crates.io/crates/yew): for frontend

Proposed structure within trustchain as:
```toml
[workspace]
members = [
    "trustchain-core",
    "trustchain-ion",
    #...    
    "trustchain-verifier-backend",
    "trustchain-verifier-frontend",
    #...
]
```

**Comment by sgreenbury** [comment]
Next steps:
- Refactor server code into a structure such as in this [example](https://auth0.com/blog/build-an-api-in-rust-with-jwt-authentication-using-actix-web/)
- Add a very basic resolution endpoint for testing the display of documents and chains in credible ([credible issue #3](https://github.com/alan-turing-institute/credible/issues/3))

**Comment by sgreenbury** [comment]
Closing in favour of combined implementation in #55.

**Comment by sgreenbury** [comment]
Next step: once #55 is closed with #96, reopen with the following todo:
- [ ] Merge `55-issuer-backend-traits-verifier-todo`
- [ ] Refine presentation request
- [ ] Add doc comments for the verifier module of `trustchain-http`
